src/implementations/vanilla/components/category-filter/category-filter-print.scss

```diff
@@ -0,0 +1,11 @@
+/*
+ * Menu component
+ * @define menu
+ */
+
+@use 'sass:map';
+@use '@ecl/theme-dev/theme';
+
+.ecl-category-filter {
+  margin: 0;
+}
```

Comment:
print version is quite basic.
we could improve it a little (use correct font, maybe remove the icons, ...)